packages/obonode/obojobo-chunks-materia/server/route-helpers.js

```diff
@@ -40,16 +39,40 @@ const ltiToolConsumer = baseUrl => ({
 	tool_consumer_instance_url: baseUrl
 })
 
-const widgetLaunchParams = (document, materiaChunkId, user, visitId, isPreview, baseUrl) => {
+const widgetLaunchParams = (document, visit, user, materiaChunkId, baseUrl) => {
 	const params = {
-		lis_outcome_service_url: `${baseUrl}/materia-lti-score-passback`,
 		lti_message_type: 'basic-lti-launch-request',
-		lis_result_sourcedid: createLisResultSourcedId(visitId, materiaChunkId),
 		lti_version: 'LTI-1p0',
-		resource_link_id: materiaChunkId
+		// materia will send scores here
+		lis_outcome_service_url: `${baseUrl}/materia-lti-score-passback`,
+		// represents the container for the user's score this widget can set/update
+		lis_result_sourcedid: `${visit.id}__${materiaChunkId}`,
+		// unique placement of this widget (take into account the module's unique placement in a course + the widget's place in the module)
+		// we are intentionally not including user id (not part of the placement) or draft_content_id (to allow updates to a module)
+		resource_link_id: `${visit.resource_link_id}__${visit.draft_id}__${materiaChunkId}`,
+		custom_chunk_id: materiaChunkId,
+		custom_visit_id: visit.id,
+		custom_draft_id: visit.draft_id,
+		custom_draft_content_id: visit.draft_content_id,
+		custom_passthrough_resource_link_id: visit.resource_link_id
+	}
+
+	// materia currently uses context_id to group scores and attempts
+	// obojobo doesn't support materia as scoreable questions yet, so the key in use here is intended to:
+	// * support materia in content pages
+	// * re lti launch will reset scores/attempts
```

Comment:
^ I don't think that's true based on the implementation of `resource_link_id` above